usl_models/usl_models/flood_ml/metastore.py

```diff
@@ -1,8 +1,42 @@
+import datetime
 from typing import Any, Sequence, TypeVar
 import urllib.parse
 
 from google.cloud import firestore  # type:ignore[attr-defined]
 
+import usl_models.flood_ml.model_params
+
+
+def write_model_metadata(
```

Comment:
Do any of the current fields hold information about the number of epochs we trained on? If not, I think that could be useful to add too.

Reply:
the epochs are part of the model params, which we write out

![Screenshot 2024-07-19 3 11 11 PM](https://github.com/user-attachments/assets/2bb2c581-ace7-4969-a947-676e4f45895a)

Reply:
oh, wait, I'm wrong -- they used to be part of the params but no longer are! I'll add that in!

Reply:
I think we're missing the field in the docstring.